multiversx_sdk/network_providers/transaction_decoder.py

```diff
@@ -89,7 +89,7 @@ def get_normal_transaction_metadata(self, transaction: TransactionOnNetwork) ->
             if len(args) == 0 and not transaction.receiver.is_smart_contract():
                 metadata.function_name = "transfer"
                 metadata.function_args = []
-                metadata.message = data_components
+                metadata.transfer_messages = [data.encode() for data in data_components]
```

Comment:
Since `transfer_messages` holds an array of bytes, and `data_components` contains hex-encoded components, apply a decoding from hex, to get the raw byes? Sorry if I'm mistaken.

Reply:
Done.